packages/expo-location/src/Location.ts

```diff
@@ -193,30 +189,31 @@ export async function geocodeAsync(
 // @needsAudit
 /**
  * Reverse geocode a location to postal address.
- * > **Note**: Using the Geocoding web api is no longer supported. Use [Place Autocomplete](https://developers.google.com/maps/documentation/places/web-service/autocomplete) instead.
+ *
+ * On Android, you must request a location permission (`Permissions.LOCATION`) from the user
+ * before geocoding can be used.
```

Comment:
```suggestion
 * On Android, you must request location permissions with [`requestForegroundPermissionsAsync`](#locationrequestforegroundpermissionsasync) before geocoding can be used.
```

Maybe we should use something like this, so it's clearer for the users how to do it.